bokeh/models/mappers.py

```diff
@@ -1,60 +1,22 @@
 from __future__ import absolute_import
 
-import numpy as np
-from six import string_types
-
 from ..plot_object import PlotObject
-from ..properties import Any, Float, Color
+from ..properties import Float, Color, Enum, Seq
+from ..enums import Palette
 from .. import palettes
 
 class ColorMapper(PlotObject):
-    ''' Base class for color mapper objects. '''
-    pass
+    """ Base class for color mapper objects. """
 
 class LinearColorMapper(ColorMapper):
+    palette = Seq(Color).accepts(Enum(Palette), lambda pal: getattr(palettes, pal))
 
-    # TODO (bev) use Array property
-    palette = Any # Array
     low = Float
     high = Float
 
-    reserve_color = Color("#ffffff") #TODO: What is the color code for transparent???
+    reserve_color = Color("#ffffff")    # TODO: What is the color code for transparent?
```

Comment:
Agreed, this is an implementation detail poking through, but it's also a bit awkward to prevent potentially inconsistent inputs from users otherwise. But we should figure out something...